src/internal/ajax/AjaxResponse.ts

```diff
@@ -81,8 +81,16 @@ export class AjaxResponse<T> {
      * The event type. This can be used to discern between different events
      * if you're using progress events with {@link includeDownloadProgress} or
      * {@link includeUploadProgress} settings in {@link AjaxConfig}.
+     *
+     * The event type consists of two parts: the {@link AjaxDirection} and the
+     * the event type. Merged with `_`, they form the `type` string. The
+     * direction can be an `upload` or a `download` direction, while an event can
+     * be `loadstart`, `progress` or `load`.
+     *
+     * `download_load` is the type of event when download has finished and the
+     * response is available.
      */
-    public readonly type = 'download_load'
+    public readonly type: string = 'download_load'
```

Comment:
This is a "fix" for the table at `/api/ajax/AjaxResponse#properties` where `type` property has had `'download_load'` TypeScript type instead of `string` type.